api/src/routing.js

```diff
@@ -94,7 +94,7 @@ if (process.argv.slice(2).includes('--allow-cors')) {
 }
 
 app.use((req, res, next) => {
-  req.id = uuid.v4();
+  req.id = uuid();
```

Comment:
I don't think you should make this change as it's not clear which version of the UUID it's generating and seems to be unrelated to the other changes.